ui/src/components/friends/add.rs

```diff
@@ -72,8 +72,16 @@ pub fn AddFriend(cx: Scope) -> Element {
     }
 
     if let Some(id) = my_id.get().clone() {
-        let mut clipboard = Clipboard::new().unwrap();
-        clipboard.set_text(id).unwrap();
+        match Clipboard::new() {
+            Ok(mut c) => {
+                if let Err(e) = c.set_text(id) {
+                    log::warn!("Unable to set text to clipboard: {e}");
+                }
+            },
```

Comment:
```suggestion
            }
```

to fix the linter issue 🔨 

<img width="806" alt="Captura de ecrã 2023-11-09, às 22 16 19" src="https://github.com/Satellite-im/Uplink/assets/29093946/323d5150-3a72-4f3e-9270-1db26193b483">